internal/scorecard/plugins/basic_tests.go

```diff
@@ -28,6 +28,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// Basic test short names
+	specBlockExists   = "specblockexists"
+	statusBlockExists = "statusblockexists"
+	writeIntoCR       = "writeintocr"
```

Comment:
I'm curious if we could use reflection and automatically use the name of the struct that implements the test?

Reply:
good suggestion, I added reflection to this now to determine the names.